cli/src/commands/describe.rs

```diff
@@ -67,35 +77,96 @@ pub(crate) fn cmd_describe(
     args: &DescribeArgs,
 ) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
-    let commit = workspace_command.resolve_single_rev(&args.revision)?;
-    workspace_command.check_rewritable([commit.id()])?;
-    let description = if args.stdin {
+    let commits = workspace_command
+        .resolve_some_revsets_default_single(&args.revisions)?
+        .into_iter()
+        .collect_vec();
```

Comment:
nit: maybe better to use `parse_union_revsets()`? I don't think the arguments order is important here, and I would rather want to edit descriptions in topological order.

BTW, `resolve_some*()` errors out if the set is empty.